packages/presentation/lib/src/pages/home_page/home.dart

```diff
@@ -24,71 +27,104 @@ class Home extends StatefulWidget {
 
 class _HomeState extends BlocScreenState<Home, HomeBloc> {
   static const _title = 'Star Movie';
+  int _bottomNavIndex = 0;
+
+  void _onBottomNavBarClick(int index) {
+    setState(() {
+      _bottomNavIndex = index;
+    });
+  }
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text(_title, style: sfProSemiBold24px),
+        centerTitle: false,
+        backgroundColor: Theme.of(context).colorScheme.background,
+        elevation: 0,
         actions: [
           IconButton(
-            onPressed: () {
-              //todo action
-            },
+            onPressed: () {},
             icon: const Icon(Icons.search),
           )
         ],
       ),
-      bottomNavigationBar: BottomNavigationBar(
+      bottomNavigationBar: buildBottomNavigationBar(),
```

Comment:
move the bottom navigator to the app's level. each screen shouldn't have it's own bottom navigation

Reply:
still actual